### PR TITLE
Update website and installation instructions

### PR DESCRIPTION
**Pull request by mkelley**
Most of the changes are reformatting, but important edits for installation instructions, where to get help, and NASA grant numbers.  Small changes to the team page, too.

### CODE REVIEW

File: website/public/index.html [hunk 1]
```diff
@@ -96,9 +92,27 @@ <h1>Learn sbpy</h1>
...
+        Discussions</a>
+      <a class="button" href="https://www.astropy.org/help.html" target="_blank">Astropy help</a>
+    </section>
+
+    <section class="whatsnext">
+      <h1>Report bugs and contribute</h1>
+      <p>Please report what you believe is a bug or a mistake using <a
+          href="https://github.com/NASA-Planetary-Science/sbpy/issues">sbpy's issue
+          tracker</a>. If you have an idea for how to improve sbpy, or would like to
+        contribute code directly, please check out the contribution guidelines.
```
Comment (jianyangli) [inline website/public/index.html:115]: add a link to the development page https://sbpy.readthedocs.io/en/latest/development/index.html for "contribution guidelines"?

File: website/public/team.html [hunk 2]
```diff
@@ -1,95 +1,91 @@
...
-	<p align="center"><a href="https://jyli.me/">Jian-Yang Li</a>, Planetary Science Institute</p>
-	<p align="center"><a href="https://www.linkedin.com/in/giannina-guzman-3965bb166/">Giannina Guzman</a>, Villanova University/Lowell Observatory</p>
-	<p align="center">Brigitta Sipőcz, University of Washington</p>
-	<p align="center">Josef Ďurech, Charles University Prague</p>
-	<p align="center">Mikael Granvik, University of Helsinki</p>
-	<p align="center">Will Grundy, Lowell Observatory</p>
-	<p align="center">Nick Moskovitz, Lowell Observatory</p>
-	<p align="center">Antti Penttilä, University of Helsinki</p>
-	<p align="center">Nalin Samarasinha, Planetary Science Institute</p>
+      <p align="center"><a href="https://www.astro.umd.edu/~msk/">
```
Comment (jianyangli) [inline website/public/team.html:67]: Your site doesn't open.

File: website/public/team.html [hunk 3]
```diff
@@ -1,95 +1,91 @@
...
+          Michael S. P. Kelley</a>, University of Maryland (Co-Lead Developer)</p>
+      <p align="center"><a href="https://jyli.me/">Jian-Yang Li</a>, Planetary Science Institute (Co-Lead
```
Comment (jianyangli) [inline website/public/team.html:69]: My site has an SSL certificate problem that I thought it had been fixed...  No need to update here, though

File: website/public/team.html [hunk 4]
```diff
@@ -1,95 +1,91 @@
...
+        Developer)</p>
+      <p align="center"><a href="https://mommermi.github.io/">
+          Michael Mommert</a>, University of St. Gallen, Switzerland (Initial Project Leader)</p>
+      <p align="center">Miguel de Val-Borro, Planetary Science Institute</p>
+      <p align="center"><a href="https://www.linkedin.com/in/giannina-guzman-3965bb166/">Giannina Guzman</a>,
```
Comment (jianyangli) [inline website/public/team.html:74]: Looks like her LinkedIn page URL is updated:
https://www.linkedin.com/in/giannina-guzm%C3%A1n-caloca-3965bb166/

Also I found she has a personal site if you want to link:
https://gianninapr.github.io/

File: website/public/team.html [hunk 5]
```diff
@@ -1,95 +1,91 @@
...
+        Villanova University/Lowell Observatory</p>
+      <p align="center">Brigitta Sipőcz, University of Washington</p>
+      <p align="center">Josef Ďurech, Charles University Prague</p>
+      <p align="center">Mikael Granvik, University of Helsinki</p>
+      <p align="center">Will Grundy, Lowell Observatory</p>
+      <p align="center">Nick Moskovitz, Lowell Observatory</p>
+      <p align="center">Antti Penttilä, University of Helsinki</p>
+      <p align="center">Nalin Samarasinha, Planetary Science Institute</p>
       <p>&nbsp; </p>
```
Comment (jianyangli) [inline website/public/team.html:83]: I can't remember clearly, but has Henry Hsieh contributed?

Also I found many on the list have websites.  Should we link all we can find?

**Review by jianyangli — APPROVED** [review]
Everything looks good to me.  My suggestions are just to make these pages better linked to other pages.  So I just approve it and you can merge whenever you are happy with it.

BTW, what HTML editor do you use?  Just curious.

File: website/public/team.html (hunk 2 above)
Comment (mkelley) [reply website/public/team.html:67]: Power outage in College Park today.

File: website/public/team.html (hunk 3 above)
Comment (mkelley) [reply website/public/team.html:69]: It works for me right now.

**Comment by mkelley** [comment]
> BTW, what HTML editor do you use? Just curious.

I'm just editing the source directly with VSCode and using the Live Server extension so that I can see the updates rendered as I save.

File: website/public/index.html (hunk 1 above)
Comment (mkelley) [reply website/public/index.html:115]: Done.

File: website/public/team.html (hunk 4 above)
Comment (mkelley) [reply website/public/team.html:74]: Fixed.

File: website/public/team.html (hunk 5 above)
Comment (mkelley) [reply website/public/team.html:83]: He's not in the contributors list https://github.com/NASA-Planetary-Science/sbpy/graphs/contributors  nor on the proposal.

I've added links for everyone.

**Pull request merged**